core/src/main/java/org/frankframework/processors/InputOutputPipeLineProcessor.java

```diff
@@ -35,7 +35,8 @@ public class InputOutputPipeLineProcessor extends AbstractPipeLineProcessor {
 	public PipeLineResult processPipeLine(PipeLine pipeLine, String messageId, Message message, PipeLineSession pipeLineSession, String firstPipe) throws PipeRunException {
 		// Reset the PipeLineSession and store the message and its id in the session
 		if (messageId == null) {
-			messageId = UUIDUtil.createSimpleUUID();
+			// This should not be touched anymore
+			messageId = MessageUtils.generateMessageId();
```

Comment:
Ik zou hier een illegalstate gooien.

Reply:
Ietwat riskant en naar mijn gevoel te ver buiten de scope van dit PR. Ik doe dat liever in een volgend PR.